modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-creation-and-sharing_{context}"]
+= Configuring a VPC to share within your AWS organization
+
+You can share subnets within a configured VPC with another AWS user account if that account is within your current AWS organization.
+
+.Procedure
+
+. From the AWS account that centrally manages your VPC, create or modify a VPC to your desired specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account will be the *VPC-owning AWS account*. 
+. In the link:https://us-east-1.console.aws.amazon.com/iamv2/[Identity and Access Management (IAM) section of the AWS console], create a new custom trust policy role for the shared VPC permissions. This role should have the following permissions:
```

Comment:
```suggestion
. In the link:https://us-east-1.console.aws.amazon.com/iamv2/[Identity and Access Management (IAM) section of the AWS console], create a custom trust policy role for the shared VPC permissions. Ensure that the role has the following permissions:
```
- Per IBM Style, "create a new" can be simplified to "create a".
- Per IBM Style, avoid modal verbs (would, should, could) if possible.